cvu/detector/yolov5/backends/yolov5_tensorrt.py

```
@@ -179,12 +198,32 @@ def _build_engine(self, onnx_weight: str, trt_engine_path: str,
             builder.max_batch_size = 1
 
             # FP16 quantization
-            if self._fp16:
+            if self._dtype == "fp16":
```

How can we showcase that ```dtype="float32"``` is also a valid option? 

I can think of three ways
- add ```elif self._dtype != "fp32": raise Exception("Invalid dtype......")```
- add a info comment here
- Have a class level protected constant ```SUPPORTED_DTYPES = ("float32", "float16", "int8")``` and check in ```__init__``` to make sure user provided dtype is valid

---

lets raise an exception if int8 and fp16 are not supported giving info about the fp32

---

yeah sure. And I think the code should also reflect that somehow. And also raise an exception in case someone gives an invalid dtype?